### PR TITLE
DL-12738 CCC changes

### PR DESCRIPTION
**Pull request by JoshMiernik**
# JIRA Story or Github Number - descriptive title

**Bug fix / New feature** (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist

 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/uk/gov/hmrc/childcarecalculatorfrontend/views/result.scala.html [hunk 1]
```diff
@@ -46,8 +46,8 @@
...
-                @messages(s"results.two.years.old.guidance.${model.location}.link.text") @messages("site.opens.new.tab")
+            <p class="govuk-body">@messages(s"results.two.years.old.guidance.${model.location}.text.before.link", appConfig.maxFreeHoursAmount) <a class="govuk-link" id="twoYearsOldHelp" target="_blank" rel="external" data-journey-click="link - click:@messages(s"results.two.years.old.guidance.${model.location}.title"):@messages(s"results.two.years.old.guidance.${model.location}.link.text", appConfig.maxFreeHoursAmount)" href="@messages(s"results.two.years.old.guidance.${model.location}.para1.help.link")">
+                @messages(s"results.two.years.old.guidance.${model.location}.link.text", appConfig.maxFreeHoursAmount) @messages("site.opens.new.tab")
```
Comment (VLukovski) [inline app/uk/gov/hmrc/childcarecalculatorfrontend/views/result.scala.html:50]: this scheme isn't being changed

File: app/uk/gov/hmrc/childcarecalculatorfrontend/views/result.scala.html [hunk 2]
```diff
@@ -88,9 +88,9 @@ <h2 class="govuk-heading-l">@messages("result.not.eligible.subheading")</h2>
...
-                        @messages(s"results.two.years.old.guidance.${model.location}.link.text") @messages("site.opens.new.tab")
+                    <p class="govuk-body">@messages(s"results.two.years.old.guidance.${model.location}.two.freehours", appConfig.maxFreeHoursAmount)</p>
+                    <p class="govuk-body">@messages(s"results.two.years.old.guidance.${model.location}.text.before.link") <a class="govuk-link" id="twoYearsOldHelp" target="_blank" rel="external" data-journey-click="link - click:@messages(s"results.two.years.old.guidance.${model.location}.title"):@messages(s"results.two.years.old.guidance.${model.location}.link.text", appConfig.maxFreeHoursAmount)" href="@messages(s"results.two.years.old.guidance.${model.location}.para1.help.link")">
+                        @messages(s"results.two.years.old.guidance.${model.location}.link.text", appConfig.maxFreeHoursAmount) @messages("site.opens.new.tab")
```
Comment (VLukovski) [inline app/uk/gov/hmrc/childcarecalculatorfrontend/views/result.scala.html:93]: this scheme isn't being changed

File: app/uk/gov/hmrc/childcarecalculatorfrontend/views/childAgedTwo.scala.html [hunk 3]
```diff
@@ -49,7 +49,7 @@
 
         @inputYesNo(
             label = HtmlContent(heading(messages("childAgedTwo.heading"))),
-            hint = Some(messages(s"childAgedTwo.guidance.$location")),
+            hint = Some(messages(s"childAgedTwo.guidance.$location", appConfig.maxFreeHoursAmount)),
```
Comment (VLukovski) [inline app/uk/gov/hmrc/childcarecalculatorfrontend/views/childAgedTwo.scala.html:52]: this page isn't being changed

File: conf/messages [hunk 4]
```diff
@@ -508,7 +508,7 @@ maxFreeHoursInfo.title = Free Childcare For Working Parents
 maxFreeHoursInfo.heading = Free Childcare For Working Parents
 maxFreeHoursInfo.you.can.get.with.colon = You can get:
 maxFreeHoursInfo.you.can.get = You can get
-maxFreeHoursInfo.you.can.get.nineTo23Months = 15 hours free childcare a week for each of your 9 to 23 months old
+maxFreeHoursInfo.you.can.get.nineTo23Months = {0} hours free childcare a week for each of your 9 to 23 months old
 maxFreeHoursInfo.you.can.get.twoYears = 15 hours free childcare a week for each of your 2 year olds
```
Comment (VLukovski) [inline conf/messages:512]: As far as I understand both 9-23 and 2yr will change to 30 in this scheme

File: conf/messages [hunk 5]
```diff
@@ -1308,7 +1308,7 @@ result.free.hours.detailPara3.link = https://www.gov.uk/find-free-early-educatio
 result.free.hours.detailPara4 = Free childcare hours stop when your child starts in reception class.
 
 result.free.childcare.working.parents.title = Free Childcare For Working Parents
-result.free.childcare.working.parents.nineTo23.Month.old = 15 hours free childcare for each of your 9 to 23 months old
+result.free.childcare.working.parents.nineTo23.Month.old = {0} hours free childcare for each of your 9 to 23 months old
 result.free.childcare.working.parents.two.year.old = 15 hours free childcare for each of your 2 year olds
```
Comment (VLukovski) [inline conf/messages:1312]: As far as I understand both 9-23 and 2yr will change to 30 in this scheme

**Review by stephen-oppong-beduh — APPROVED** [review]

**Pull request merged**